.travis.yml

```diff
@@ -6,7 +6,7 @@ dist: trusty
 
 env:
   matrix:
-    - PYTHON=2.7 TESTS=true PACKAGES="python-blosc futures faulthandler lz4"
+    - PYTHON=2.7 TESTS=true PACKAGES="python-blosc futures faulthandler lz4 tornado=4.5"
```

Comment:
I did not encounter this specific bug with python 2.7 and tornado 4.5, probably because of the `if 'asyncio' in sys.modules` check.